hack/builder/Dockerfile

```diff
@@ -113,6 +113,7 @@ RUN set -x && \
 
 RUN set -x && \
     source /etc/profile.d/gimme.sh && \
+    go install -v mvdan.cc/gofumpt@v0.6.0 && \
```

Comment:
AFAIU, this tool is a [part](https://golangci-lint.run/usage/linters/#gofumpt) of `golangci-lint`.
Will it be installed twice after this change?

Reply:
Per my understanding, it is embedded into the `golangci-lint` but not performing the auto-formatting.
It is installed here in order to allow the auto-formatting, not related to running the linter validation.